src/Parser/Lexer/Package.idr

```diff
@@ -99,6 +99,7 @@ andop = is '&' <+> is '&'
 rawTokens : TokenMap Token
 rawTokens =
   [ (comment, Comment . drop 2)
+  , (blockComment, Comment . shrink 2)
```

Comment:
I suppose this lacks a small amount of fidelity since the only useful meaning of the string within the comment would probably account for the fact that a block comment can start with a bracket followed by any number of dashes, but seeing as how 2 characters is the minimum at the start and end that doesn't feel too critical.

Reply:
Since we neither require a matching dash count on the end-of-comment token, nor strip excess leading dashes from eol comments, I don't feel this needs to change.

Parser/Lexer/Source.idr doesn't drop OR shrink for comments (of either type), but it also doesn't attempt to preserve their content in the tokens.